### PR TITLE
Basic version of a python script that reads a HDF5 file

### PR DESCRIPTION
**Pull request by privts**

### CODE REVIEW

File: src/bin/hdf5PythonScript/hdf5_example.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+'''
+
+import pandas
+
+ts = pandas.read_csv("data.csv", delim_whitespace=True)
+
+# Following line was used to create the HDF5 file in teh first place
+# ts.to_hdf("data.hdf5", "table")
+
+ts2 = pandas.read_hdf("data.hdf5", "table")
```
Comment (privts) [inline src/bin/hdf5PythonScript/hdf5_example.py]: A short example comment

**Comment by marayl** [comment]
The Travis build failed for some unrelated reason. I have kicked off another.

File: src/bin/hdf5PythonScript/hdf5_example.py (hunk 1 above)
Comment (marayl) [inline src/bin/hdf5PythonScript/hdf5_example.py]: Not necessary now, but we can add CMake directives to install scripts to bin directory if need be.

**Comment by marayl** [comment]
We can also add a top-level data directory for data files that need to be installed.

**Pull request merged**